tests/scripts/depends-hashes.pl

```diff
@@ -67,27 +71,29 @@ sub abort {
     exit 1;
 }
 
-for my $hash (@hashes) {
+for my $hash_config (@hash_configs) {
     system( "cp $config_h.bak $config_h" ) and die "$config_h not restored\n";
     system( "make clean" ) and die;
 
     print "\n******************************************\n";
-    print "* Testing hash option: $hash\n";
+    print "* Testing hash options: ", join(', ', @$hash_config), "\n";
     print "******************************************\n";
-    $ENV{MBEDTLS_TEST_CONFIGURATION} = "-$hash";
 
-    system( "scripts/config.py $hash" )
-        and abort "Failed to $hash\n";
+    for my $hash (@$hash_config) {
+        $ENV{MBEDTLS_TEST_CONFIGURATION} = "-$hash";
+        system( "scripts/config.py $hash" )
+            and abort "Failed to $hash\n";
+    }
 
     for my $opt (@ssl) {
         system( "scripts/config.py unset $opt" )
             and abort "Failed to disable $opt\n";
     }
 
     system( "CFLAGS='-Werror -Wall -Wextra' make lib" )
-        and abort "Failed to build lib: $hash\n";
-    system( "cd tests && make" ) and abort "Failed to build tests: $hash\n";
-    system( "make test" ) and abort "Failed test suite: $hash\n";
+        and abort "Failed to build lib: @$hash_config\n";
```

Comment:
`@$hash_config` results in something like `unset MBEDTLS_SHA512_Cunset MBEDTLS_SHA384_C`. Better make this `join(', ', @$hash_config)` as above (put this in a local variable).